components/style/servo/media_queries.rs

```diff
@@ -202,6 +203,12 @@ impl Device {
     pub fn safe_area_insets(&self) -> SideOffsets2D<f32, CSSPixel> {
         SideOffsets2D::zero()
     }
+
+    /// Returns true if the given MIME type is supported
+    pub fn is_supported_mime_type(&self, mime_type: &str) -> bool {
+        // TODO: being a valid MIME type doesn't imply that it's supported.
+        mime_type.parse::<Mime>().is_ok()
```

Comment:
This should probably try to do something correct even if it results in code duplication. Maybe use the list from the networking crate, with a comment that it should be kept in sync:

- "image/x-icon"
- mime::IMAGE_BMP
- mime::IMAGE_GIF
- "image/webp"
- mime::IMAGE_PNG
- mime::IMAGE_JPEG